docs/core/containers/publish-configuration.md

```diff
@@ -411,11 +411,25 @@ dotnet publish -p ContainerUser=root
 
 ### Default container labels
 
-Labels are often used to provide consistent metadata on container images. This package provides some default labels to encourage better maintainability of the generated images.
-
-- `org.opencontainers.image.created` is set to the ISO 8601 format of the current value of <xref:System.DateTime.UtcNow?displayProperty=nameWithType>.
-
-For more information, see [Implement conventional labels on top of existing label infrastructure](https://github.com/dotnet/sdk-container-builds/issues/96).
+Labels are often used to provide consistent metadata on container images. This package provides some default labels to encourage better maintainability of the generated images. All default label generation can be disabled by setting `ContainerGenerateLabels` to `false`. In addition, each default label has an  individual enablement flag that can be set to `false` to disable that specific label.
```

Comment:
> This package provides...

Which package?

Reply:
Ah, bad verbiage from when the container tools were primarily delivered by the Microsoft.Net.Build.Containers NuGet package. Will patch this up to remove the reference.